### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by offa**
Some minor code cleanup.

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry --> No changelog necessary
- [x] Please describe what you did
- [x] ~~Link to relevant issues in GitHub or Jira~~
- [x] ~~Link to relevant pull requests, esp. upstream and downstream changes~~
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/java/edu/hm/hafner/analysis/LineRangeList.java [hunk 1]
```diff
@@ -77,9 +78,7 @@ public LineRangeList(final Collection<LineRange> copy) {
     public LineRangeList(final LineRange... initialElements) {
         this(initialElements.length * 4); // guess
 
-        for (LineRange element : initialElements) {
-            add(element);
-        }
+        this.addAll(Arrays.asList(initialElements));
```
Comment (uhafner) [inline src/main/java/edu/hm/hafner/analysis/LineRangeList.java:81]:
```suggestion
        addAll(Arrays.asList(initialElements));
```

File: src/main/java/edu/hm/hafner/analysis/parser/CadenceIncisiveParser.java [hunk 2]
```diff
@@ -98,10 +97,6 @@ else if (matcher.group(21) != null) {
         if (fileName == null) {
             return Optional.empty();
         }
-        if (fileName.startsWith(SLASH)) {
-            return builder.setFileName(fileName).setLineStart(lineNumber).setCategory(category)
-                    .setMessage(message).setSeverity(priority).buildOptional();
-        }
```
Comment (uhafner) [inline src/main/java/edu/hm/hafner/analysis/parser/CadenceIncisiveParser.java:104]: I wonder what the original author intended to do here... But anyway, this makes no sense and all tests are green. So it is save to remove.

**Review by uhafner — APPROVED** [review]
Just some nitpicking...

**Comment by offa** [comment]
Thanks, PR updated.

**Pull request merged**